source_code/main_mcu/src/defines.h

```diff
@@ -27,6 +27,9 @@
 
 /* Defines */
 #define AUX_MCU_MESSAGE_REPLY_TIMEOUT_MS    1500
+#define TOTP_SECRET_MAX_LEN                 32
```

Comment:
can you put these in nodemgmt.h instead, as they're linked to the DB format?

Reply:
I was doing that initially but decided on this file instead since TOTP_SECRET_MAX_LEN, SERVICE_NAME_MAX_LEN & LOGIN_NAME_MAX_LEN is used in BOTH nodemgmt.h AND comms_hid_defines.h. 

Nodemgmt.h already has an include for "defines.h". I could move them all to Nodemgmt.h but then I would have a circular include (defines.h would have to include nodemgmt.h and nodemgmt.h already includes defines.h).

What do you suggest?

Reply:
https://github.com/mooltipass/minible/blob/master/source_code/main_mcu/src/NODEMGMT/nodemgmt_defines.h then :)

Reply:
Done